ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -70,14 +67,18 @@ class GraphsContainer extends React.Component {
     }
 
     if (prevProps.timeRange !== timeRange && this.state.dataPoint) {
-      this.setState({
-        dataPoint: null,
-        showTooltip: false,
-        lockTooltip: false,
-      });
+      this.onUpdate();
```

Comment:
Did you change this because of the `react/no-did-update-set-state` rule?

Reply:
But since you've already done it, can you give this function a clearer name?

Reply:
Sorry, my mistake! :(

Sure, I've changed to `resetDataPoint()`.